docs/source/exploring.rst

```diff
@@ -3,7 +3,11 @@
 Explore the API using Postman
 =============================
 
-To discover the API interactively, the Chrome-Extension Postman_ is a suitable solution.
+To discover the API interactively, the Postman_ is recommended.
+
+.. admonition:: Note
+
+    The Chrome-Extension version of Postman is deprecated and it is recommended to use the native apps available instead.
```

Comment:
"native apps" -> "native app"

Reply:
I used plural referring to the different apps for each OS. Should I still change it? @tisto